src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -1830,11 +1819,12 @@ public void storeMetric(GenericMetric metric) {
             metric.getClusterName(),
             metric.getMetricDomain(),
             metric.getMetricType(),
-            metric.getTs().toString(HOURLY_FORMATTER),
+            metric.getTs().toString(TIME_BUCKET_FORMATTER).substring(0, 11) + "0",
             metric.getHost(),
             metric.getMetricScope(),
             metric.getMetricName(),
-            metric.getTs().toDate(),
+            METRIC_TS_FORMATTER.parseDateTime(
+                metric.getTs().toString(METRIC_TS_FORMATTER).substring(0, 15) + "0").toDate(),
```

Comment:
nit-pick: -flooring the timestamp to the minute (or the ten-minute above) can be done in smarter ways using javax.time,  instead of text formatting.